packages/world/src/ObjectType.sol

```diff
@@ -714,6 +714,11 @@ library ObjectTypeLib {
   }
 
   // Specialized getters
+
+  function isStation(ObjectType self) internal pure returns (bool) {
```

Comment:
What are we using this for?

Reply:
not really using it, I thought it was good to have it, but can remove for simplicity